Assets/MRTK/SDK/Experimental/Drawstring/Scripts/ElasticHandleLogic.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Physics.Experimental
```

Comment:
```suggestion
namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
```

Reply:
"Experimental" is a prefix in namespaces (see:

https://github.com/microsoft/MixedRealityToolkit-Unity/search?p=4&q=experimental&unscoped_q=experimental)

Reply:
We had a chat about this at some point in the past and landed on prefixing instead of suffixing. There are pros and cons to each obviously